pyHalo/PresetModels/external.py

```diff
@@ -187,43 +186,20 @@ def DMFromEmulator(z_lens, z_source, concentration_model_subhalos='BOSE2016', kw
     from pyHalo.preset_models import preset_model_from_name
 
     # FIRST WE CREATE AN INSTANCE OF PYHALO, WHICH SETS THE COSMOLOGY
-    pyhalo = pyHalo(z_lens, z_source, kwargs_cosmo)
+    pyhalo = pyHalo(z_lens, z_source)
     # WE ALSO SPECIFY THE GEOMETRY OF THE RENDERING VOLUME
     geometry = Geometry(pyhalo.cosmology, z_lens, z_source,
                         cone_opening_angle_arcsec, geometry_type)
 
-    # SET THE CONCENTRATION-MASS RELATION FOR SUBHALOS AND FIELD HALOS
-    kwargs_concentration_model_subhalos['cosmo'] = pyhalo.astropy_cosmo
-
-    model_subhalos, kwargs_mc_subs = preset_concentration_models(concentration_model_subhalos,
-                                                                 kwargs_concentration_model_subhalos)
-
-
-    concentration_model_subhalos = model_subhalos(log_mc = log_mc, **kwargs_mc_subs)
-    c_host = concentration_model_subhalos.nfw_concentration(10 ** log_m_host, z_lens)
-
-    # SET THE TRUNCATION RADIUS FOR SUBHALOS AND FIELD HALOS
-    kwargs_truncation_model_subhalos['lens_cosmo'] = pyhalo.lens_cosmo
-    kwargs_truncation_model_subhalos['c_host'] = c_host
-
-    model_subhalos, kwargs_trunc_subs = truncation_models(truncation_model_subhalos)
-    kwargs_trunc_subs.update(kwargs_truncation_model_subhalos)
-    truncation_model_subhalos = model_subhalos(**kwargs_trunc_subs)
-
-    kwargs_halo_model = {'truncation_model_subhalos': truncation_model_subhalos,
-                         'concentration_model_subhalos': concentration_model_subhalos,
-                         'kwargs_density_profile': {}}
-
-    halos_LOS = preset_model_from_name(preset_model_los)(**emulator_kwargs)
-    rendering_classes = halos_LOS.rendering_classes
-
     # get lens_cosmo class from class containing LOS objects; note that this will work even if there are no LOS halos
+    halos_LOS = preset_model_from_name(preset_model_los)(**emulator_kwargs)
     lens_cosmo = halos_LOS.lens_cosmo
 
-    kpc_per_arcsec_at_z = lens_cosmo.cosmo.kpc_proper_per_asec(z_lens)
+    #kpc_per_arcsec_at_z = lens_cosmo.cosmo.kpc_proper_per_asec(z_lens)
 
-    h = lens_cosmo.h
-    rhoc_zlens = lens_cosmo._nfw_param.rhoc_z(z_lens)
+    #h = lens_cosmo.h
+    h = 0.7
```

Comment:
instead of hard-coding h, access the value of little h corresponding to the assumed cosmology: 

h = pyhalo.lens_cosmo.h